cirq/ionq/ionq_client_test.py

```diff
@@ -427,3 +427,68 @@ def test_ionq_client_delete_job_retry(mock_put):
     )
     client.delete_job('job_id')
     assert mock_put.call_count == 2
+
+
+@mock.patch('requests.get')
+def test_ionq_client_get_current_calibrations(mock_get):
+    mock_get.return_value.ok = True
+    mock_get.return_value.json.return_value = {'foo': 'bar'}
+    client = ionq.ionq_client._IonQClient(remote_host='http://example.com', api_key='to_my_heart')
+    response = client.get_current_calibration()
+    assert response == {'foo': 'bar'}
+
+    expected_headers = {'Authorization': 'apiKey to_my_heart', 'Content-Type': 'application/json'}
+    mock_get.assert_called_with(
+        'http://example.com/v0.1/calibrations/current', headers=expected_headers
+    )
+
+
+@mock.patch('requests.get')
+def test_ionq_client_get_current_calibration_unauthorized(mock_get):
+    mock_get.return_value.ok = False
+    mock_get.return_value.status_code = requests.codes.unauthorized
+
+    client = ionq.ionq_client._IonQClient(
```

Comment:
Does `default_target` argument affect the outcome of the test or can be omitted?

Reply:
Yep, good catch.  Removed.  Though I will ask you later how there can be calibrations for different targets in the long run :)